tests/test_ujson.py

```diff
@@ -275,6 +275,53 @@ def test_encode_indent(test_input):
     assert output == json.dumps(obj, indent=4)
 
 
+def test_indent_types():
+    # Python json allows indent to be an int, str, or None.
+    # Check that ujson handles this as well.
+    data = [1, 2, 3]
+    output0a = ujson.encode(data)
+    output0b = ujson.encode(data, indent=None)
+    output0c = ujson.encode(data, indent="")
+    output0d = ujson.encode(data, indent=0)
```

Comment:
These (and also a negative ints) should actually produce a different output than default and `indent=None`, namely inserting newlines but no indentation. However, that bug has been present for longer (#317), and if you don't want to tackle it, it could be done at a later time.